packages/infra/lib/ihe-stack.ts

```diff
@@ -168,11 +185,21 @@ export class IHEStack extends Stack {
       envVars: {
         ...(props.config.lambdasSentryDSN ? { SENTRY_DSN: props.config.lambdasSentryDSN } : {}),
       },
-      vpc: props.vpc,
-      alarmSnsAction: props.alarmAction,
+      vpc,
+      alarmSnsAction,
+      version: props.version,
     });
 
-    const xcpdResource = api.root.addResource("xcpd");
-    xcpdResource.addMethod("ANY", new apig.LambdaIntegration(patientDiscoveryLambda));
+    apiResource.addMethod("ANY", new apig.LambdaIntegration(patientDiscoveryLambda));
   }
 }
+
+function setupSlackNotifSnsTopic(stack: Stack, config: EnvConfig): SnsAction | undefined {
```

Comment:
nit: at this point, will be the 4th `setupSlackNotifSnsTopic` in the infra. Might we worth a refactor?

Reply:
I only see two `setupSlackNotifSnsTopic` on the OSS infra, one on API and another one on IHE's. I also checked for code that uses SNS `Topic`, and nothing else beyond those two. What are you referring to?

As for API and IHE, those are different functions:
- API: provisions the infra
- IHE: gets a reference to a previously provisioned infra

Reply:
I am trying to figure out what I was looking at, since your totally right, and there are only 2. With respect to the 2 that exist, what you said makes sense .